docs/user_manual/orders.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: "11. Orders"
+section: 11
+---
+
+Once a pool is ready for sequencing, there maybe delay to batch multiple pools
+for a single flow cell and a pool maybe placed in multiple lanes. To facilitate
```

Comment:
The first sentence is a bit complex... What do you think about `Orders are used to track which pools are ready to be sequenced, how much sequencing needs to be done for a given pool, and whether that required sequencing has been completed.` to replace both sentences in this paragraph?